AutomatticTracks/src/sharedTest/kotlin/com/automattic/android/tracks/fakes/FakeDataProvider.kt

```diff
@@ -12,9 +11,9 @@ class FakeDataProvider(
     override val releaseName: String = "testReleaseName",
     override val locale: Locale? = Locale.US,
     override val enableCrashLoggingLogs: Boolean = true,
-    override val toDropIfLastException: CrashLoggingException? = null,
     var user: CrashLoggingUser? = testUser1,
     var userHasOptOut: Boolean = false,
+    var shouldDropException: (String, String, String) -> Boolean = { _: String, _: String, _: String -> false },
```

Comment:
`(String, String, String)` doesn't look very pretty but it's used only internally. I can suggest either `typealias` or `inline class` or leaving it as it is. WDYT @malinajirka ?

Reply:
I think it's fine for a FakeDataProvider. We might want to consider replacing the fake provider with a mocked one in the future, but it's not a priority and perhaps not even a good idea.